README.md

```diff
@@ -1,7 +1,7 @@
 # Polybar
 
 [![Build Status](https://travis-ci.org/jaagr/polybar.svg?branch=master)](https://travis-ci.org/jaagr/polybar)
-[![Coverage Status](https://coveralls.io/repos/github/jaagr/polybar/badge.svg?branch=master)](https://coveralls.io/github/jaagr/polybar?branch=master)
+[![Coverage Status](https://codecov.io/gh/jaagr/polybar/branch/master/graph/badge.svg)](https://codecov.io/gh/jaagr/polybar/branch/master/graph/badge.svg)
```

Comment:
Please make the badge link directly to the codecov page. After that, I think this is ready to merge